docs/DeveloperGuide.md

```diff
@@ -713,3 +713,38 @@ testers are expected to do more *exploratory* testing.
 
    2. **Test case:** `edit 10 n/John Doe p/98765432`<br>
       **Expected:** No client is edited. Error details shown in the status message.
+
+### Viewing a Client
+
+1. **Viewing a client's details**
+
+   1. **Prerequisites:** Ensure that client to view exists.
+
+   1. **Test case:** `view 1`<br>
+      **Expected:** Popup window of the clients details displayed. Details of client is showed in the window
+
+   1. **Test case:** `view -1`<br>
+      **Expected:** No client is viewed. Error details shown in the status message.
+
```

Comment:
i see that for delete-client and delete-car, there are blanket cases like this:

 1. **Other incorrect delete commands to try:** `del-car`, `del-car x`, `...` (where x is larger than the list size)<br>
      **Expected:** Similar to previous.
      
      not required, but can consider changing it for the view/find and putting it in?
      
      otherwise, i'll just merge and reupload new DG by tomorrow